website/database.py

```diff
@@ -617,9 +666,11 @@ def delete_adventure(self, adventure_id):
     def store_adventure(self, adventure):
         """Store an adventure."""
         ADVENTURES.create(adventure)
+        self.update_public_adventure_filters_indexes(adventure)
 
     def update_adventure(self, adventure_id, adventure):
         ADVENTURES.update({"id": adventure_id}, adventure)
+        self.update_public_adventure_filters_indexes(adventure)
```

Comment:
Same here.

Reply:
By the way in the indexes, you probably need to remove tags/levels if those arrays got changed.